internal/controller/storageclaim_controller.go

```diff
@@ -51,9 +51,12 @@ import (
 const (
 	storageClaimFinalizer  = "storageclaim.ocs.openshift.io"
 	storageClaimAnnotation = "ocs.openshift.io/storageclaim"
+	keyRotationAnnotation  = "keyrotation.csiaddons.openshift.io/schedule"
 
 	pvClusterIDIndexName  = "index:persistentVolumeClusterID"
 	vscClusterIDIndexName = "index:volumeSnapshotContentCSIDriver"
+
+	cronScheduleWeekly = "@weekly"
```

Comment:
It would be good if you could find a suitable place to add it globally because it is being used in many places now.

Reply:
How about I take it in a separate PR?

Reply:
In this PR, you can move and use it. In a different PR, you can make changes to the existing code.

Reply:
Done

Reply:
yes LGTM.